### PR TITLE
added Parasara Sridhar Duggirala and Junier Oliva as faculty to UNC-CS

### PR DESCRIPTION
**Pull request by psduggirala**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -9959,6 +9960,7 @@ Paolo Tonella,Università della Svizzera italiana,http://www.inf.usi.ch/faculty/
 Parag Chaudhuri,IIT Bombay,https://www.cse.iitb.ac.in/~paragc,kOkSgaMAAAAJ
 Parag Singla,IIT Delhi,http://www.cse.iitd.ac.in/~parags,V49BsgMAAAAJ
 Parampalli Udaya,University of Melbourne,http://people.eng.unimelb.edu.au/udaya,dT9UeiAAAAAJ
+Parasara Sridhar Duggirala,University of North Carolina,https://www.linkedin.com/in/parasarasridhar/,00S6A4IAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:9963]: Fix home page: http://cs.unc.edu/~psd/

File: csrankings.csv [hunk 2]
```diff
@@ -6869,6 +6869,7 @@ Jung-Hwan Oh,University of North Texas,http://www.cse.unt.edu/~jhoh,NOSCHOLARPAG
 Junghoo Cho,University of California - Los Angeles,http://www.cs.ucla.edu/~cho,DSLquxEAAAAJ
 Junhui Deng,Tsinghua University,https://dsa.cs.tsinghua.edu.cn,NOSCHOLARPAGE
 Junichi Suzuki,University of Massachusetts Boston,https://www.umb.edu/academics/csm/faculty_staff/jun_suzuki,4oVjDuoAAAAJ
+Junier B. Oliva, University of North Carolina,https://cs.unc.edu/people/junier-oliva/,CqH_t6MAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:6872]: Please use https://sites.google.com/cs.unc.edu/joliva/home.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 2 above)
Comment (psduggirala) [reply csrankings.csv:6872]: Done.

File: csrankings.csv (hunk 1 above)
Comment (psduggirala) [reply csrankings.csv:9963]: Done.

**Comment by emeryberger** [comment]
Thanks!

**Pull request merged**